herbert_core/classes/MPIFramework/@MessagesFilebased/MessagesFilebased.m

```diff
@@ -26,7 +26,7 @@
     end
     properties(Constant=true,Access=protected,Hidden=true)
         % The number, used in folder name and Job ID to define folder migration.
-        folder_migration_shift =  0011100001;
+        folder_migration_shift =  0000000001;
```

Comment:
What is the relevance of this bitmask change?

Reply:
that's have actually picked up from another ticket -- nothing to do with config 
this one is e0f1b72 only

Reply:
To be fair, I've started this one from #192 by mistake, though all should go in the same place.